Examples/SimpleGaussian/display_input.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+# display_input.py
+# Minimal routine for running the Example code and displaying input image.
+# The plot_image() routine is not specific to this example.
+from gaussian_utils import run_gaussian
+from plot_util import plot_image
+
+input_img, _ = run_gaussian()
+plot_image(input_img)
```

Comment:
Why were these couple of lines chosen to be in a separate file as opposed to being inline?

Reply:
I was trying to avoid repeating myself. If I put that code inline, I'd have to replicate the two import statements. Also, I'm not sure the :align: center would work with inline code, but I guess it should. (Of course, it's not working now, anyway).

Reply:
Also, I keep knocking around the idea of making this object oriented, and having the standard interfaces of display_input() / display_output() would work well in a class. Not sure how well this would play with Sphinx, though, and I haven't tried coding it up, yet.

Reply:
I think it would be easier in the docs file.

Later we can refine run_gaussian to run_example(name_of_example, args..) so that it can be used by multiple example.

Reply:
Okay, I'll go ahead and make that change. Is okay if I also pull plot_image() up into the parent directory where all Examples can access it? Assuming I can get the import to work, that is.

Reply:
yes put plot_image and run_guassian on the same file in the parent directory.

Reply:
But run_gaussian is very specific to SimpleGaussian. So I guess we'll make this change, verify that it works, and then genericize `run_gaussian()` to `run_example()` like you said. But I'll still need to figure out how to dynamically import the `SimpleGaussian.main()` based on the name. But I think that's doable - I guess we'll find out! 😏